config/slips.yaml

```diff
@@ -0,0 +1,445 @@
+
+# All the following detections are turned on by default
+# Turn them off by adding any of the following detections to the disabled_detections list
+
+# ARP_SCAN, ARP_OUTSIDE_LOCALNET, UNSOLICITED_ARP, MITM_ARP_ATTACK,
+# YOUNG_DOMAIN, MULTIPLE_SSH_VERSIONS, DIFFERENT_LOCALNET,
+# DEVICE_CHANGING_IP, NON_HTTP_PORT_80_CONNECTION, NON_SSL_PORT_443_CONNECTION,
+# WEIRD_HTTP_METHOD, INCOMPATIBLE_CN, DGA_NXDOMAINS, DNS_WITHOUT_CONNECTION,
+# PASTEBIN_DOWNLOAD, CONNECTION_WITHOUT_DNS, DNS_ARPA_SCAN, UNKNOWN_PORT,
+# PASSWORD_GUESSING, HORIZONTAL_PORT_SCAN, CONNECTION_TO_PRIVATE_IP, GRE_TUNNEL,
+# VERTICAL_PORT_SCAN, SSH_SUCCESSFUL, LONG_CONNECTION, SELF_SIGNED_CERTIFICATE,
+# MULTIPLE_RECONNECTION_ATTEMPTS, CONNECTION_TO_MULTIPLE_PORTS, HIGH_ENTROPY_DNS_ANSWER,
+# INVALID_DNS_RESOLUTION, PORT_0_CONNECTION, MALICIOUS_JA3, MALICIOUS_JA3S,
+# DATA_UPLOAD, BAD_SMTP_LOGIN, SMTP_LOGIN_BRUTEFORCE, MALICIOUS_SSL_CERT,
+# MALICIOUS_FLOW, SUSPICIOUS_USER_AGENT, EMPTY_CONNECTIONS, INCOMPATIBLE_USER_AGENT,
+# EXECUTABLE_MIME_TYPE, MULTIPLE_USER_AGENT, HTTP_TRAFFIC, MALICIOUS_JARM,
+# NETWORK_GPS_LOCATION_LEAKED, ICMP_TIMESTAMP_SCAN, ICMP_ADDRESS_SCAN,
+#  ICMP_ADDRESS_MASK_SCAN, DHCP_SCAN, MALICIOUS_IP_FROM_P2P_NETWORK, P2P_REPORT,
+#  COMMAND_AND_CONTROL_CHANNEL, THREAT_INTELLIGENCE_BLACKLISTED_ASN,
+#  THREAT_INTELLIGENCE_BLACKLISTED_IP, THREAT_INTELLIGENCE_BLACKLISTED_DOMAIN,
+# MALICIOUS_DOWNLOADED_FILE, MALICIOUS_URL
+
+# disabled_detections = [THREAT_INTELLIGENCE_BLACKLISTED_IP, CONNECTION_TO_PRIVATE_IP]
+   disabled_detections : []
+
+####################
+# [10] Docker created files settings
+Docker:
+# ID and group id of the user who started to docker container
+# the purpose of using them is to change the ownership of the docker created files to be able to rwx the files from outside docker too, for example the files in the output/ dir
+   UID : 0
+   GID : 0
+
+####################
+
+Profiling:
+
+# [11] CPU profiling
+
+# enable cpu profiling [yes,no]
+   cpu_profiler_enable : no
+
+# Available options are [dev,live]
+# dev for deterministic profiling. this will give precise information about the CPU usage
+# throughout the program runtime. This module cannot give live updates
+# live mode is for sampling data stream. To track the function stack in real time. it is accessible from web interface
+   cpu_profiler_mode : dev
+
+# profile all subprocesses in dev mode [yes,no].
+   cpu_profiler_multiprocess : yes
+
+# set number of tracer entries (dev mode only)
+   cpu_profiler_dev_mode_entries : 1000000
+
+# set maximum output lines (live mode only)
+   cpu_profiler_output_limit : 20
+
+# set the wait time between sampling sequences in seconds (live mode only)
+   cpu_profiler_sampling_interval : 20
+
+# [12] Memory Profiling
+
+# enable memory profiling [yes,no]
+   memory_profiler_enable : no
+
+# set profiling mode [dev,live]
+   memory_profiler_mode : live
+
+# profile all subprocesses [yes,no]
+   memory_profiler_multiprocess : yes
+
```

Comment:
in the latest develop, there's a parameter here for the web interface port, slips willl complain when it's missing